index.html

```diff
@@ -0,0 +1,48 @@
+        </p>
+      </header>
+      <section id="form_header">
+        <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D07cade4b-b92b-4c5b-86ac-526257555814%26adobe_mc%3DMCMID%253D53422390496372272074315764205492827937%257CMCAID%253D2EEECDC20515F78C-600007FE4F268601%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1574813265%26ivid%3D07cade4b-b92b-4c5b-86ac-526257555814"></a><img src="image/intuit.png" alt="intuit" id="intuit">
+          <div id="icons">  
+            <a href="https://www.mint.com" class="h_links"><span><img src="image/mint.svg" alt="mint" class="icon">mint</span></a>
+            <a href="https://quickbooks.intuit.com/" class="h_links"><span><img src="image/quickbooks.svg" alt="quickbooks" class="icon">quickbooks</span></a>
+            <a href="https://turbotax.intuit.com/" class="h_links"><span><img src="image/tt.svg" alt="tt" class="icon">turbotax</span></a>
+          </div>      
+      </section>
+      <form>
+        <h3>Create an Intuit account</h3>
+        <p>One account for everything Intuit, including Mint.<br>
+        <a href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account">Learn more</a><br>
+        Already have an account? <a href="https://accounts.intuit.com/index.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D082e7ed1-15b6-4d9c-a5b0-c4cfc0b2a48b%26adobe_mc%3DMCMID%253D17380633903390345294292432376599021936%257CMCAID%253D2EEECCB7051596DF-600008078EA69FD9%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1574803839%26ivid%3D082e7ed1-15b6-4d9c-a5b0-c4cfc0b2a48b">Sign in</a></p>
+        <fieldset>
+          <label for="email" >Email address</label>
+          <input type="email" name="email" id ="email">
+          <label for="phone">Phone (recommended)</label>
+          <input type="tel" name="phone" id ="phone">
+          <label id="calls" for="phone">Standard call, messaging or data rates may apply.</label>
+          <label for="password">Password</label>
+          <input type="password" name="password" id ="password">
+        </fieldset>
+        <button id="create_acc" class="but"><img src="image/lock.png" alt="lock" class="lock">Create Account</button>
```

Comment:
Not showing either.

![img](https://user-images.githubusercontent.com/31304943/69757500-71607900-115d-11ea-88ea-f3777ea823d0.png)